Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/BaseDAO.java

```diff
@@ -163,6 +164,17 @@ public void evict(T baseBean) {
         evictObject(baseBean);
     }
 
+    /**
+     * Merge given bean object.
+     *
+     * @param baseBean
+     *            bean to Merge
+     */
+    @CheckReturnValue
+    public T merge(T baseBean) {
+        return mergeObject(baseBean);
+    }
```

Comment:
🛈 In order to get a bean into the current Hibernate session, it is necessary to continue working with the return value, because Hibernate may insert an (other) proxy object in between. This is documented with the annotation `@CheckReturnValue`.

This is different from the other functions:

```java
myDao.save(myBean); // ✓
myDao.refresh(myBean); // ✓
// but:
myDao.merge(myBean); // ✗ no effect
// correct:
myBean = myDao.merge(myBean); // ✓

```